apps/cli/src/config.ts

```diff
@@ -3,6 +3,72 @@ import os from "os";
 import { extname } from "path";
 import { TomlPrimitive, parse as parseToml } from "smol-toml";
 
+/**
+ * Typed Errors
+ */
+export class InvalidBuilderError extends Error {
+    constructor(builder: any) {
```

Comment:
don't use any, use `string`

Reply:
When using `string`, the IDE suggest to call it likle `new InvalidBuilderError(true as unknown as string)`, which is weird for me.

What's the best approach in those cases?